BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -9,6 +11,9 @@ namespace BricksAndHearts.UnitTests.ControllerTests.Admin;
 
 public class AdminControllerTests : AdminControllerTestsBase
 {
+    private static readonly IAdminService adminService = A.Fake<IAdminService>();
+    private readonly AdminController _underTest = new(null!, null!, adminService);

```

Comment:
Why have these been moved out of `AdminControllerTestsBase`?

Reply:
There was a potential issue with a static fake admin service being reused for every test - Ben suggested moving the setup to the constructor instead (see the conversation under Frank's suggestions above, I've also screenshotted it and put it here for convenience). I'm not actually certain that I've now put them in the intended place - I think maybe they should be in each individual test? - but that's why they've moved out of AdminControllerTestsBase.
![image](https://user-images.githubusercontent.com/108676120/181192127-6d149105-0940-4362-a0e7-637d04bdfa7b.png)

Reply:
I think he meant to put in the constructor of `AdminControllerTestsBase`, like:
```
public class AdminControllerTestsBase: ControllerTestsBase
{
    protected readonly IAdminService AdminService;
    protected readonly AdminController UnderTest;

    protected AdminControllerTestsBase()
    {
        AdminService = A.Fake<IAdminService>();
        UnderTest = new(null!, null!, AdminService);
    }
}
```
otherwise it doesn't seem the tests base is really serving any purpose. @BenRamchandani have I understood what you were suggesting?

Reply:
(and same for any other controller tests base files)